### PR TITLE
refactor: project structure

### PR DESCRIPTION
**Pull request by kastrubait**

### CODE REVIEW

**Review by coriander31415 — APPROVED** [review]

File: rslang/src/app/components/dot-menu/dot-menu.component.html [hunk 1]
```diff
@@ -5,29 +5,29 @@
...
     </button>
     <button mat-menu-item [routerLink]="['statistics']" routerLinkActive="active">
       <mat-icon>graphic_eq</mat-icon>
-      <span>Статистика</span>
+      <span>statistics</span>
     </button>
     <button mat-menu-item [routerLink]="['auth']" routerLinkActive="active">
       <mat-icon>account_circle</mat-icon>
-      <span>Войти</span>
+      <span>Sign In</span>
```
Comment (Anntol) [inline rslang/src/app/components/dot-menu/dot-menu.component.html:24]: Some menu items are Upper cased, others are Lower cased. Some items are still in Russian. Why?

Comment (kastrubait) [reply rslang/src/app/components/dot-menu/dot-menu.component.html:24]: I have not edited this menu. Just moved to fit the new structure
 In the nav-menu, I added links to the application pages, logo, translation into English. Styling will be completed after adding the main page. Therefore, it is different.

**Review by whattablackhole — APPROVED** [review]

File: rslang/src/app/app.module.ts [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
 import { BrowserModule } from '@angular/platform-browser';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';
-import { AppRoutingModule } from './app-routing.module';
-import { MaterialModule } from './material/material.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { MatDialogModule } from '@angular/material/dialog';
```
Comment (Nikitzu) [inline rslang/src/app/app.module.ts]: I do not understand why we have so mane Mat modules in App.

**Review by Nikitzu — CHANGES_REQUESTED** [review]

File: rslang/src/app/app.module.ts (hunk 2 above)
Comment (kastrubait) [reply rslang/src/app/app.module.ts]: modules are imported into shared

**Review by Nikitzu — APPROVED** [review]

**Pull request merged**